source/gui/settingsDialogs.py

```diff
@@ -1135,75 +1170,104 @@ def makeSettings(self, settingsSizer):
+		self.headingsCheckBox.SetValue(config.conf["documentFormatting"]["reportHeadings"])
+
 		# Translators: This is the label for a checkbox in the
 		# document formatting settings dialog.
-		self.headingsCheckBox=wx.CheckBox(self,wx.NewId(),label=_("Report &headings"))
-		self.headingsCheckBox.SetValue(config.conf["documentFormatting"]["reportHeadings"])
-		settingsSizer.Add(self.headingsCheckBox,border=10,flag=wx.BOTTOM)
+		self.linksCheckBox=otherGroup.addItem(wx.CheckBox(scrolledItems,label=_("Lin&ks")))
+		self.linksCheckBox.SetValue(config.conf["documentFormatting"]["reportLinks"])
+
 		# Translators: This is the label for a checkbox in the
 		# document formatting settings dialog.
-		self.listsCheckBox=wx.CheckBox(self,wx.NewId(),label=_("Report &lists"))
+		self.listsCheckBox=otherGroup.addItem(wx.CheckBox(scrolledItems,label=_("&Lists")))
 		self.listsCheckBox.SetValue(config.conf["documentFormatting"]["reportLists"])
-		settingsSizer.Add(self.listsCheckBox,border=10,flag=wx.BOTTOM)
+
 		# Translators: This is the label for a checkbox in the
 		# document formatting settings dialog.
-		self.blockQuotesCheckBox=wx.CheckBox(self,wx.NewId(),label=_("Report block &quotes"))
+		self.blockQuotesCheckBox=otherGroup.addItem(wx.CheckBox(scrolledItems,label=_("Block &quotes")))
 		self.blockQuotesCheckBox.SetValue(config.conf["documentFormatting"]["reportBlockQuotes"])
-		settingsSizer.Add(self.blockQuotesCheckBox,border=10,flag=wx.BOTTOM)
+		
 		# Translators: This is the label for a checkbox in the
 		# document formatting settings dialog.
-		self.landmarksCheckBox=wx.CheckBox(self,wx.NewId(),label=_("Report lan&dmarks"))
+		self.landmarksCheckBox=otherGroup.addItem(wx.CheckBox(scrolledItems,label=_("Lan&dmarks")))
 		self.landmarksCheckBox.SetValue(config.conf["documentFormatting"]["reportLandmarks"])
-		settingsSizer.Add(self.landmarksCheckBox,border=10,flag=wx.BOTTOM)
+		
 		# Translators: This is the label for a checkbox in the
 		# document formatting settings dialog.
-		item=self.framesCheckBox=wx.CheckBox(self,label=_("Report fra&mes"))
-		item.Value=config.conf["documentFormatting"]["reportFrames"]
-		settingsSizer.Add(item,border=10,flag=wx.BOTTOM)
+		self.framesCheckBox=otherGroup.addItem(wx.CheckBox(scrolledItems,label=_("Fra&mes")))
+		self.framesCheckBox.Value=config.conf["documentFormatting"]["reportFrames"]
+		
 		# Translators: This is the label for a checkbox in the
 		# document formatting settings dialog.
-		item=self.clickableCheckBox=wx.CheckBox(self,label=_("Report if &clickable"))
-		item.Value=config.conf["documentFormatting"]["reportClickable"]
-		settingsSizer.Add(item,border=10,flag=wx.BOTTOM)
+		self.clickableCheckBox=otherGroup.addItem(wx.CheckBox(scrolledItems,label=_("&Clickable")))
+		self.clickableCheckBox.Value=config.conf["documentFormatting"]["reportClickable"]
+
+		sHelper.addItem(scrolledItems)
+		scrolledItems.Layout()
+		scrolledItems.SetupScrolling()

```

Comment:
This is also called above when creating the panel. Are both these calls necessary?